app/models/journey.rb

```diff
@@ -0,0 +1,42 @@
+class Journey < ApplicationRecord
+  belongs_to :move
+  belongs_to :supplier
+  belongs_to :from_location, class_name: 'Location'
```

Comment:
Could we delegate to the move location or is it simpler/more flexible to duplicate these?

Reply:
ah nope we can't delegate - a move from A->B may be composed of multiple journeys A -> C ->D -> B. 
So locations definitely need to be stored at both the Move level and at the Journey level.